explainer.md

```diff
@@ -60,18 +60,20 @@ The entire painting of the element is captured, including things which appear ou
 
 https://user-images.githubusercontent.com/93594/141118353-d62d19a1-0964-4fa0-880f-bdde656ce899.mp4
 
-The element is captured without the effects (such as opacity and filters) from parent elements. Effects on the element itself are baked into the image. However, the element is captured without transforms, as those transforms are reapplied later.
+The element is captured without the effects (such as opacity and filters) from parent elements. Effects on the element itself are baked into the image. However, the element is captured without transforms, as those transforms are reapplied later. The root is always captured as a single image, with the other transition elements removed (similar to how compositing works today).
```

Comment:
> similar to how compositing works today

I'm not sure what that means, maybe remove it?

Reply:
This was meant to indicate that removing the elements is similar to how browsers composite elements into separate textures today. The alternate text was something like : "The root is always captured as a single image, with the other transition elements removed (as-if those elements have opacity 0 in the image)."

Is that better? Also @jakearchibald on this.

Reply:
We can take this in a separate PR. Wanted to get Jake's thoughts too since he added this.